### ISSUE TITLE
Add captions display for audio if/when available

### ISSUE THREAD

**Issue by joncameron** [samvera-labs/ramp#180]
# Description
WCAG 1.2.2 is to provide captions for pre-recorded audio content. Currently, captions are displayed if available for video but not for audio in Avalon.

How can Avalon provide caption data for audio-only content?

Transcript component can be used to render this out as accessible text alongside the media resource. Caption file content could also be display inside the player by adjusting the aspect ratio and showing the caption text inside the media player element.

IU has some [nice guidelines](https://kb.iu.edu/d/avhj?_gl=1*1989x1t*_ga*MTQzODUxNTI1Ni4xNjc3NTk1NzE1*_ga_61CH0D2DQW*MTY4ODA1MDgxMi44Ni4xLjE2ODgwNTA4NTAuMjIuMC4w) for how to handle this case. They indicate that transcripts should be used in lieu of captions for audio-only components.

# Definition of Done
- [ ] Investigation of options for providing accessible text for audio-only content in Avalon
- [ ]

**Comment by joncameron** [comment]
One proposed solution would be casting caption files as transcripts, if not explicit transcripts are available for an audio masterfile but a caption file is present.

**Comment by elynema** [comment]
A good place to look for models may be established podcast sites (here's an [article](https://castos.com/podcast-transcription/) that provides some links). Most of them have a button for a transcript with some examples of opening it in a new window or popup. This matches with the idea of linking to transcript within the audio embedded player, I believe. The article suggests that downloadable transcripts aren't as accessible. 

This [w3.org example](https://www.w3.org/WAI/perspective-videos/captions/#transcript) has a 'show transcript' button in the player right next to the caption button. That might be something similar to what we could do.

**Comment by joncameron** [comment]
Handling transcript display for embedded audio materials is something that we'll want to look into in the future. We currently meet this at IU by providing transcripts or caption files as transcripts in Media Collections Online.
